mongo-statefulset/README.md

```diff
@@ -68,65 +73,61 @@ $oc new-project mongodb-statefulset
 
 ### Adding QuickStart Templates
 
-The template containing the MongoDB StatefulSet is located within the _mongo-statefulset_ folder.
-
-If you have cloned the repository to your local machine, navigate to the *mongo-statefulset* folder and execute the following command to add the templates to the OpenShift project.
+In each folder defined in the "Template Files" section, enter the folder and create each template by running
 
 ```
-$ oc create -f mongodb-statefulset-replication.yaml
+$oc create -f .
```

Comment:
rather than create the template, and then process it later, can we just process the file directly with `oc process -f`? We're moving more to that pattern elsewhere as well. This would just remove this intermediary step.